guides/instrumenting-a-module.md

```diff
@@ -68,10 +13,10 @@ Wherever possible, it is preferred that `ao.instrument(...)` is used, however
 there are some alternatives, which we'll explore later. The `ao.instrument(...)`
 function takes four arguments:
 
-The signature is `ao.instrument (span, fn, config, callback)`.
+The signature is `ao.instrument (span, runner, config, callback)`.
 
 - `span` is either the string name of the span to be created or a function that returns the span to use.
```

Comment:
do you mean "a function that returns the name to use"?

Reply:
i added (see below) where it is explained.